src/common/markdown.ts

```diff
@@ -0,0 +1,18 @@
+import MarkdownIt from 'markdown-it';
+import hljs from 'highlight.js';
+
+const markdown: MarkdownIt = MarkdownIt({
+  html: true,
+  linkify: true,
+  typographer: true,
+  highlight: function (str, lang) {
+    if (lang && hljs.getLanguage(lang)) {
+      try {
+        return hljs.highlight(str, { language: lang }).value;
+      } catch (__) {}
+    }
+    return '';
```

Comment:
Should we return the empty string or the original string `str`?

Reply:
Yep! Missed that one